src/containers/Viewer.jsx

```diff
@@ -104,10 +105,16 @@ export class Viewer extends Component {
 }
 
 const mapStateToProps = (state, ownProps) => {
-  let set = state.photos.map(photo => photo._id)
+  let photos = []
+  if (ownProps.router.routes[1].path === 'albums') { // photos from an album
```

Comment:
I think the information of the current route should be somewhere in the state, as we know do in Files. But I'm ok with leaving that as it is at the moment.